packages/create-app/templates/default-app/packages/backend/src/index.ts

```diff
@@ -46,6 +47,7 @@ async function main() {
   const configs = await loadBackendConfig();
   const configReader = ConfigReader.fromConfigs(configs);
   const createEnv = makeCreateEnv(configs);
+  await ensureDatabaseExists(configReader.getConfig('backend.database'), 'backstage_plugin_catalog', 'backstage_plugin_auth')
```

Comment:
hm i wonder if linting is disabled inside the templates or something - this should be linebroken like the main code above is

Reply:
Fixed.